modules/videosipgw/VideoSIPGW.js

```diff
@@ -0,0 +1,144 @@
+import { getLogger } from 'jitsi-meet-logger';
+const logger = getLogger(__filename);
+
+import JitsiVideoSIPGWSession from './JitsiVideoSIPGWSession';
+import * as Constants from './VideoSIPGWConstants';
+import XMPPEvents from '../../service/xmpp/XMPPEvents';
+
+/**
+ * Main video SIP GW handler. Stores references of all created sessions.
+ */
+export default class VideoSIPGW {
+
+    /**
+     * Creates new handler.
+     *
+     * @param {ChatRoom} chatRoom - Tha chat room to handle.
+     */
+    constructor(chatRoom) {
+        this.chatRoom = chatRoom;
+        this.eventEmitter = chatRoom.eventEmitter;
+        logger.info('creating VideoSIPGW');
+        this.sessions = {};
+
+        this.sessionStateChangeListener = this.sessionStateChanged.bind(this);
+
+        chatRoom.addPresenceListener('jibri-sip-status',
```

Comment:
I see we never remove presence listener. Can you add some comment which explains why ? Like for example "VideoSIPGW, JitsiConference and ChatRoom are not reusable and no more than one VideoSIPGW can be created per JitsiConference, so we don't bother to cleanup".